counsel.el

```diff
@@ -4918,6 +4918,36 @@ An extra action allows to switch to the process buffer."
 ;; `counsel-slime-repl-history' within
 ;; `counsel--browse-history-action'.
 
+;;** `counsel-esh-dir-history'
```

Comment:
Shouldn't this be added near `counsel-esh-history`?

Reply:
Yes, it's next to it in the outline:

```
;;** `counsel-esh-history'
;;** `counsel-esh-dir-history'
```

However it's true that `counsel-slime-repl-history` is in between but unfortunately it does not have it's own header, guess that's why I missed it. Anyway, I've moved a bit closer. There's now `counsel-shell-history` in between but it didn't feel right to take it out of the `counsel-esh-history` header, at least not without the maintainers' consent. Happy to reorg stuff even more if I get a blessing :)

Reply:
I guess the simplest solution is to add `counsel-esh-dir-history` right before the `counsel-esh-history` heading, no?

Reply:
He, I have to confess that this idea crossed my mind but I didn't want to be impolite by taking that first place ;). It's changed now.